app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    current_user.present?
+  end
+
+  def log_out
+    session.delete(:user_id)
```

Comment:
```suggestion
    session.delete :user_id
```
hạn chế dùng ngoặc trong ruby